.github/workflows/reusable-build-test-release.yml

```diff
@@ -2433,9 +2277,9 @@ jobs:
               echo "run-publish=false" >> "$GITHUB_OUTPUT"
           fi
       - name: exit without publish
-        if: ${{ steps.check.outputs.run-publish == 'false' || ( github.event.action == 'labeled' && github.event.label.name == 'preserve_infra' ) }}
+        if: ${{ steps.check.outputs.run-publish == 'false' || ( github.event.action == 'labeled') }}
         run: |
-          echo "Some test job failed or Workflow has triggered on preserve_infra label."
+          echo "some test job failed."
```

Comment:
Let's be a bit more dev oriented here and say something like "please expand check step to see which job has failed pre-publish step".

Reply:
changed